api/swagger.yaml

```diff
@@ -77,43 +90,337 @@ tags:
-      | restrictionWarningEvent         | ●        |         |          | ●     | ●     |       | `PPAR`          | A warning event that may be sent when a restriction is violated.
-      | trafficWarningEvent             | ●        |         |          |       | ●     |       | `PPAR`          | A warning event that may be sent to warn about a traffic condition on a `Location` or a `Route`.
-      | linkTripsEvent                  |          |         |          | ●     |       |       | `PPAR`          | Event to be used to link two `Trip`s together.
-      | pickupTimeWindowStartEvent      |          |         | ●        |       |       |       | `PP__`          | Marks the start of a time window for the pickup of a `Shipment`. Only applicable in `planned` or `projected` life cycle phases.
-      | pickupTimeWindowEndEvent        |          |         | ●        |       |       |       | `PP__`          | Marks the end of a time window for the pickup of a `Shipment`. Only applicable in `planned` or `projected` life cycle phases.
-      | deliveryTimeWindowStartEvent    |          |         | ●        |       |       |       | `PP__`          | Marks the start of a time window for the delivery of a `Shipment`. Only applicable in `planned` or `projected` life cycle phases.
-      | deliveryTimeWindowEndEvent      |          |         | ●        |       |       |       | `PP__`          | Marks the end of a time window for the delivery of a `Shipment`. Only applicable in `planned` or `projected` life cycle phases.
-      | constraintViolationEvent        | ●        |         | ●        | ●     |       |       | `PPAR`          | Can be generated if a constraint is violated.
-      | cancelEvent                     | ●        | ●       | ●        | ●     | ●     | ●     | `__AR`          | Cancel one or more `Event`. To be canceled event(s) should be set as `involvedObject(s)`. Since events can only be created and not deleted nor updated, a `cancelEvent` is needed when you need to change or remove an event.
-      | ignoreEvent                     | ●        | ●       | ●        | ●     | ●     | ●     | `__AR`          | Ignore one or more `Event`. To be ignored event(s) should be set as `involvedObject(s)`.
-      | associateWithActorEvent         | ●        | ●       | ●        | ●     | ●     | ●     | `_PAR`          | Associate an entity with an `Actor`.
+      are the core of this API. Events link all static objects together. The sections below document the available event types. Each section contains
+      a table that summarizes to which entities the event can be applied and a table that denotes in which life cycle phases the event is available.
+
+      ## Update events
+
+      Update events typically send an update to the state of an entity. These kind of events are only sensible in the `actual` and `realized` life cycle
```

Comment:
To me it's not entirely clear what update events are. An update to the state of an entity entails putting that new entity. Is there a reason to describe this in an event?